src/main.rs

```diff
@@ -71,140 +76,6 @@ fn cert_entries() -> Result<Vec<CertTableEntry>> {
     }
 }
 
```

Comment:
Do we want to keep this functions in main.rs? Or maybe create and IOCTL.rs? Not necessary, just a suggestion.